syntax = "proto3";

service Store {
  rpc GetPrice(GetPriceRequest) returns (GetPriceResponse) {}
  rpc Sell(SellRequest) returns (SellResponse) {}
  rpc End(Empty) returns (ResponseEnd) {}
}

message GetPriceRequest {}

message GetPriceResponse {
  string price = 1;
}

message SellRequest {
    int32 order_id = 1;
}

message SellResponse {
    string status = 1;
}

message Empty {}

message ResponseEnd {
    int32 revenue = 1;
    int32 pendencies = 2;
}
